### ISSUE TITLE
[sass] Using Keyframe Mixin (Compass) produces a Fatal

### ISSUE THREAD

**Issue by wernerdweight** [sasstools/sass-lint#797]
**What version of Sass Lint are you using?**
v1.8.2

**Please include any relevant parts of your configuration**
I don't think it's relevant, but here it is:

``` yml
rules:
  empty-line-between-blocks: 0
```

**What did you do? Please include the actual source code causing the issue.**

``` sass
+keyframes(loadMoreSpinner)
    0%
        +transform(rotate(0deg) scale(1))
    50%
        +transform(rotate(180deg) scale(1.1))
    100%
        +transform(rotate(360deg) scale(1))
```

**What did you expect to happen?**
No linting errors (or maybe some, but not a Fatal).

**What actually happened? Please include any error messages given to you by Sass Lint.**

``` bash
dev/sass/wd/_animations.sass
  2:1  error  Please check validity of the block starting from line #2  Fatal
```

**If you're using a IDE plugin have you tried the CLI too?**
Yes, same error.

**Comment by DanPurdy** [comment]
This is a bug with the AST you need to report it at https://github.com/tonyganch/gonzales-pe

**Comment by raphaelfabeni** [comment]
I have the same error here using this:

```sass
@keyframes shadow
  0%,
  20%
    background-color: rgba(0, 0, 0, 0)
    transform: scale(0)
```

😢

**Comment by DanPurdy** [comment]
Would you mind adding it to the linked issue on the gonzales repo above your comment @raphaelfabeni would help a lot when we look into those issues there. Thanks

**Comment by raphaelfabeni** [comment]
@DanPurdy I did that! Thanks for the fast answer.

**Comment by bgriffith** [comment]
This is fixed as of `v1.11.1`
